css/main.css

```diff
@@ -319,44 +222,21 @@ textarea {
     }
 }
 
-/*=============================================================
-Dev Challenge CSS - Version 2.0
-===============================================================
-*/
+/* ==========================================================================
+   Author's custom styles
+   ========================================================================== */
 
 body {
-    width:960px;
-    padding:0 20px 0 20px;
-
+    margin-top:25px;
 }
 
-
 header {
-    width:100%;
     margin:25px 0 10px 0;
-    display:block;
-    height:75px;
-
+    height:100px;
     border-bottom: solid 20px grey;
-    /*
-    border-width:10px 10px 10px 10px;
-
-
-    /*border-image: url(../img/horizonal_border.png);
-
-    border-image: url(../img/horizonal_border.png) 25 30 10 20 repeat stretch;
-
-
-    border-image: url(../img/horizontal_border.png) 25 30 10 20 repeat stretch;
-    */
-
     -webkit-border-image:url(../img/horizontal_border.png) 30 30 round; /* Safari 3.1-5 */
     -o-border-image:url(../img/horizontal_border.png) 30 30 round; /* Opera 11-12.1 */
     border-image:url(../img/horizontal_border.png) 30 30 round;
-
-     /* CSS for bottom border - should be able to use an image for the line effect
-    border-bottom: url(horizonal_border.png): 15px repeat-x;
-    */
 }
 
 header img {
```

Comment:
Watch out with selectors this broad. This is affecting .share-icons img as well, though you were able to patch it. If you're specifically targeting the logo image, class and select it more precisely.